src/lightning_app/frontend/panel/panel_frontend.py

```diff
@@ -69,20 +84,15 @@ def configure_layout(self):
 
         app = L.LightningApp(LitApp())
 
-    You can start the Lightning server with Panel autoreload by setting the `PANEL_AUTORELOAD`
-    environment variable to 'yes': `AUTORELOAD=yes lightning run app app_basic.py`.
-
-    Args:
-        entry_point: A pure function or the path to a .py or .ipynb file.
-        The function must be a pure function that contains your Panel code.
-        The function can optionally accept an `AppStateWatcher` argument.
+    Start the Lightning server with `lightning run app app_basic.py`.
 
-    Raises:
-        TypeError: Raised if the entry_point is a class method
+    For development you can get Panel autoreload by setting the `PANEL_AUTORELOAD`
+    environment variable to 'yes', i.e. run
+    `PANEL_AUTORELOAD=yes lightning run app app_basic.py`
```

Comment:
Note: single ticks make it look something like italic, double ticks will make it an inline code piece.